evap/evaluation/tests/test_models.py

```diff
@@ -854,7 +854,7 @@ def test_missing_email_address():
         """
         user = baker.make(UserProfile, email=None)
         template = EmailTemplate.objects.get(name=EmailTemplate.STUDENT_REMINDER)
-        template.send_to_user(user, {}, {}, False, None)
+        template.send_to_user(user, {}, {}, False, [])
```

Comment:
We should consider making most of those keyword-only. Maybe in a follow up for first semester students? Not the most fun thing to do though